File: Entities/Base/Response.cs
using Core.Entities;

namespace Entities.Base
{
    public class Response : IEntity
    {
        public int Id { get; set; }
        public int QuestionId { get; set; }
        public int FormId { get; set; }
        public int UserId { get; set; }
    }
}

File: WebUI/UI/src/app/datas/models/base/response.ts
export class Response {
  constructor(questionId: number, formId: number, userId: number) {
    this.questionId = questionId;
    this.formId = formId;
    this.userId = userId;
  }

  id: number;
  questionId: number;
  formId: number;
  userId: number;
}
